src/app/legacy/components/Promo/index.test.jsx

```diff
@@ -77,3 +81,19 @@ describe('Promo component - Image', () => {
     });
   });
 });
+
+describe('Promo component - Timestamp', () => {
+  it('should render timestamp in string format', () => {
+    const { container } = render(<Fixture timestamp={stringTimestamp} />, {
+      service: 'kyrgyz',
+    });
+    expect(container).toBeInTheDocument('5 саат мурда');
+  });
+  it('should render timestamp in epoch format', () => {
+    const { container } = render(<Fixture timestamp={unixTimestamp} />, {
+      service: 'serbian',
+      variant: 'cyr',
+    });
+    expect(container).toBeInTheDocument('Пре 4 сата');
+  });
```

Comment:
We might also want to add a test for dates in the past (in which a past date can just be hardcoded and we expect the date to render), as well as tests for the relative dates.

Perhaps we could have a describe within  `Promo component - Timestamp` for past dates, and another for relative dates?

Also, could we default the service to pidgin / afrique / mundo so that that the test result is slightly easier to read?

Reply:
As suggested, Mundo has been used now to make this easier